jobs/update-legal-filings/k8s/templates/bc.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  labels:
+    app: ${NAME}
+  name: ${NAME}-build
+objects:
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    name: ${NAME}
+    labels:
+      app: ${NAME}
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    name: ${NAME}
+    labels:
+      app: ${NAME}
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: ${NAME}:${OUTPUT_IMAGE_TAG}
+    resources:
+      limits:
+        cpu: ${CPU_LIMIT}
+        memory: ${MEMORY_LIMIT}
+      requests:
+        cpu: ${CPU_REQUEST}
+        memory: ${MEMORY_REQUEST}
+    runPolicy: Serial
+    source:
+      contextDir: ${SOURCE_CONTEXT_DIR}
+      git:
+        ref: ${GIT_REF}
+        uri: ${GIT_REPO_URL}
+      dockerfile: |
+        FROM docker-remote.artifacts.developer.gov.bc.ca/python:3.8.5-buster
+        USER root
+
+        # Create working directory
+        RUN mkdir /opt/app-root && chmod 755 /opt/app-root
+        WORKDIR /opt/app-root
+
+        # Install the requirements
+        COPY ./requirements.txt .
+
+        #RUN pip install --upgrade pip
+        RUN pip install pip==20.1.1
```

Comment:
Again, is special PIP version required?

Reply:
There were issue to use new version pip to build the application. Need someone to verify it we can use other version of pip.

Reply:
If the current script works then this conversation can be resolved, but we should have a tech debt ticket to verify/upgrade the pip version.